bin/mail/2016-11-15-hoc-global-minecraft/send-real-emails

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# 750550 teachers
+../send-to-mailing-list --limit 751000 2016-11-15-hoc-global-minecraft ./teachers.csv
```

Comment:
Do you send to the 750_550 list or send to a regenerated (updated list)? If the latter, I think we should increase the limit, as we are seeing O(2K) `HocSignup2016` signups per day. And sorry for not thinking of this in prior PRs.